packages/actor-bindings-aggregator-factory-average/test/AverageAggregator-test.ts

```diff
@@ -13,6 +13,27 @@ async function runAggregator(aggregator: IBindingsAggregator, input: RDF.Binding
   return aggregator.result();
 }
 
+async function createAggregator({ expressionEvaluatorFactory, context, distinct }: {
+  expressionEvaluatorFactory: IExpressionEvaluatorFactory;
+  context: IActionContext;
+  distinct: boolean;
+}): Promise<AverageAggregator> {
+  return new AverageAggregator(
+    await expressionEvaluatorFactory.createEvaluator(makeAggregate('avg', distinct).expression, context),
+    distinct,
+    <ITermFunction> await expressionEvaluatorFactory.createFunction({
```

Comment:
Isn't there a way to return `ITermFunction` from `createFunction` so the cast is not needed?